src/main/java/uk/gov/companieshouse/controller/AppealController.java

```diff
@@ -65,6 +72,11 @@ public AppealController(AppealService appealService) {
 
             return ResponseEntity.created(location).body(id);
 
+        } catch (AppealReasonException appealReasonException) {
+            LOGGER.error(appealReasonException.getMessage() +
+                " for company number {}, penalty reference {} and user id {}",
+                    companyId, penaltyReference, userId, appealReasonException);
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
         } catch (Exception ex) {
```

Comment:
I know it was already like this, but could we make this more specific than all Exceptions. https://stackify.com/common-mistakes-handling-java-exception/